.github/dependabot.yml

```diff
@@ -9,7 +9,7 @@ updates:
     schedule:
       interval: "weekly"
     reviewers:
-      - "adidahiya"
+      - "palantir/blueprint"
```

Comment:
Matched the syntax for teams listed here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#reviewers